docker/docker-compose.dev.yml

```diff
@@ -0,0 +1,81 @@
+name: lilypad
+services:
+  chain:
+    image: offchainlabs/nitro-node:v2.3.4-b4cc111
+    container_name: chain
+    ports:
+      - 8547:8547
+      - 8548:8548
+    command:
+      [
+        "--init.dev-init",
+        "--init.dev-init-address",
+        $ADMIN_ADDRESS,
+        "--node.dangerous.no-l1-listener",
+        "--node.parent-chain-reader.enable=false",
+        "--parent-chain.id=1337",
+        "--chain.id=412346",
+        "--node.sequencer",
+        "--execution.sequencer.enable",
+        "--node.dangerous.no-sequencer-coordinator",
+        "--node.staker.enable=false",
+        "--init.empty=false",
+        "--http.port",
+        "8547",
+        "--http.addr",
+        "0.0.0.0",
+        "--ws.port",
+        "8548",
+        "--ws.addr",
+        "0.0.0.0",
+      ]
+    volumes:
+      - ../data/chain:/home/user/.arbitrum
+  solver:
+    image: ghcr.io/lilypad-tech/solver
+    container_name: solver
+    depends_on:
+      - chain
+    build:
+      context: ..
+      dockerfile: ./docker/solver/Dockerfile
+      args:
+        - network=dev
+    extra_hosts:
+      - "localhost:host-gateway"
+    environment:
+      - DOPPLER_TOKEN=${DOPPLER_TOKEN_SOLVER}
+    ports:
+      - 8080:8080
+  job-creator:
+    image: ghcr.io/lilypad-tech/job-creator
+    container_name: job-creator
+    depends_on:
+      - solver
+    build:
+      context: ..
+      dockerfile: ./docker/job-creator/Dockerfile
+      args:
+        - network=dev
+    extra_hosts:
+      - "localhost:host-gateway"
+    environment:
+      - DOPPLER_TOKEN=${DOPPLER_TOKEN_JOB_CREATOR}
+  resource-provider:
+    image: ghcr.io/lilypad-tech/resource-provider
+    container_name: resource-provider
+    depends_on:
+      - chain
+      - solver
+    build:
+      context: ..
+      dockerfile: ./docker/resource-provider/Dockerfile
+      args:
+        - NETWORK=dev
+    extra_hosts:
+      - "localhost:host-gateway"
+    volumes:
+      - ../data/bacalhau:/root/.bacalhau
+      - /var/run/docker.sock:/var/run/docker.sock
```

Comment:
I think we may need to mount a folder for the results too, something like `./data/bacalhau:/tmp/lilypad/data`

When I run the `onchain` job (`./stack run-cowsay-onchain`), it reported the job to have run successfully but I could not find any folder with the results locally.

Reply:
right! I was playing around with this and forgot to double check. i've switched it to mount `/tmp/lilypad/data` instead. thanks!